### DISCUSSION TITLE
ALPR support?

### DISCUSSION THREAD

**Discussion by ajkelsey** [glossyio/traffic-monitor#38] [General]
Nice project. I wish it came along a year ago and it would have saved me some trouble building my speed camera. 😆 Does your project support ALPR?

**Comment by glossyio** [comment]
Hey, @ajkelsey!  Thank you for your interest. It's been a work in progress for some time but finally getting more formalized.  

ALPR, short answer is Yes, but it isn't integrated out-of-the-box. The object detection uses [Frigate](https://github.com/blakeblackshear/frigate), which has [many discussions around using ALPR](https://github.com/search?q=repo%3Ablakeblackshear%2Ffrigate+alpr&type=discussions).  I think the RPi5 can handle LOCAL ALPR with just the CPU. However, I was using my home server with CodeProject.ai to do it for my unit by sending the `car` detections to my server, parsing for the license plate and collecting that info.  [Frigate+](https://docs.frigate.video/plus/) has models that you can also use to make it even easier!  Those are per-user and provide personalization.

**Comment by glossyio** [comment]
Frigate officially added LPR support in [v0.16](https://github.com/blakeblackshear/frigate/releases/tag/v0.16.0). Might require Frigate+ unless we include a separate detection model: Refer to the [License Plate Recognition documentation](https://docs.frigate.video/configuration/license_plate_recognition).
